webapp/src/main/java/org/opentestsystem/rdw/admin/model/GroupQuery.java

```diff
@@ -0,0 +1,65 @@
+package org.opentestsystem.rdw.admin.model;
+
+import java.util.Set;
+
+public class GroupQuery {
+    private int schoolId;
```

Comment:
I think it depends on how robust our api needs to be if we are only building what the UI needs?  If it needs to be robust and we validate, then yes this should be non-primitive...  If not, and we trust the UI won't send bad values, then we are okay here I think.